.kapetanios/postsubmit.yaml

```diff
@@ -111,6 +111,7 @@ spec:
         type: PROJECT
     steps:
       - name: web-build
+        timeout: 20m
```

Comment:
Timeout should be set for a job, not a step. ww

Reply:
Oh, I see.
Thank you!